src/components/Header/Header.tsx

```diff
@@ -1,73 +1,145 @@
-		DEFAULT_HEADER.homeLinkProps
-	);
-	const [serviceTitle, setServiceTitle] = useState(DEFAULT_HEADER.serviceTitle);
-	const [operatorLogo, setOperatorLogo] = useState(DEFAULT_HEADER.operatorLogo);
-	const [quickAccessItems, setQuickAccessItems] = useState<Array<any>>([]);
+  useEffect(
+    () => {
+      if (!header) {
+        return;
+      }
+      setBrandTop(header.brandTop || DEFAULT_HEADER.brandTop);
+      setHomeLinkProps(header.homeLinkProps || DEFAULT_HEADER.homeLinkProps);
+      setServiceTitle(header.serviceTitle || DEFAULT_HEADER.serviceTitle);
+      setOperatorLogo(header.operatorLogo || DEFAULT_HEADER.operatorLogo);
+    },
+    [header]
+  );
 
-	useEffect(
-		() => {
-			if (!header) {
-				return;
-			}
-			setBrandTop(header.brandTop || DEFAULT_HEADER.brandTop);
-			setHomeLinkProps(header.homeLinkProps || DEFAULT_HEADER.homeLinkProps);
-			setServiceTitle(header.serviceTitle || DEFAULT_HEADER.serviceTitle);
-			setOperatorLogo(header.operatorLogo || DEFAULT_HEADER.operatorLogo);
-		},
-		[header]
-	);
+  useEffect(
+    () => {
+      const others = header?.quickAccessItems || [];
+      setQuickAccessItems([
+        ...others,
+        {
+          iconId: 'fr-icon-lock-line',
+          buttonProps: {
+            onClick: handleOidcAuth,
+          },
+          text: getAuthLabel(isAuthenticated),
+        },
+      ]);
+    },
+    [isAuthenticated, handleOidcAuth, header]
+  );
 
-	useEffect(
-		() => {
-			const others = header?.quickAccessItems || [];
-			setQuickAccessItems([
-				...others,
-				{
-					iconId: 'fr-icon-lock-line',
-					buttonProps: {
-						onClick: handleOidcAuth,
-					},
-					text: getAuthLabel(isAuthenticated),
-				},
-			]);
-		},
-		[isAuthenticated, handleOidcAuth, header]
-	);
+  return (
+    <header role="banner" className="fr-header" id="header">
+      <div className="fr-header__body">
+        <div className="fr-container">
+          <div className="fr-header__body-row">
+            <div className="fr-header__brand fr-enlarge-link">
+              <div className="fr-header__brand-top">
+                <div className="fr-header__logo">
+                  <p className="fr-logo">
+                    {<ConvertContent content={brandTop as any} />}
+                  </p>
+                </div>
+                <div className="fr-header__operator">
+                  <img className="fr-responsive-img" style={{ maxWidth: '3.5rem' }} src={operatorLogo?.imgUrl} alt={operatorLogo?.alt} />
+                </div>
+                <div className="fr-header__navbar">
+                  <button className="fr-btn--menu fr-btn" data-fr-opened="false" aria-controls="modal-577" aria-haspopup="menu" id="button-578" title="Menu">
+                    Menu
+                  </button>
+                </div>
+              </div>
+              <div className="fr-header__service">
+                <a href={homeLinkProps.href} title={`Accueil - ${homeLinkProps.title} - République Française`}>
```

Comment:
To aide lisibility of the code, I recommend destructuring variables outside of the `return` function. 
i.e: 
`const {href, title} = homeLinkProps;`
`<a href={href} title={"Accueil - ${title} - République Française"}>`
We generally try to destructure variables where possible, and where it aids lisibility.